src/static/styles/partials/_dashboard.css

```
@@ -211,3 +410,270 @@
+  background: var(--accent);
+  color: var(--accent-ink);
+  border: 1px solid var(--accent);
+  box-shadow: var(--shadow-xs);
+}
+
+.dashboard-header-button.primary:hover {
+  background: var(--accent-hover);
+  border-color: var(--accent-hover);
+}
+
+.dashboard-hero {
```

Stylelint (17.7.0)

[error] Duplicate selector ".dashboard-hero", first used at line 1 (no-duplicate-selectors)

(no-duplicate-selectors)